public/app/features/plugins/datasource_srv.ts

```diff
@@ -114,15 +105,19 @@ export class DatasourceSrv implements DataSourceService {
       };
     }
 
-    return this.settingsMapByUid[nameOrUid] ?? this.settingsMapByName[nameOrUid];
+    return this.settingsMapByUid[nameOrUid] ?? this.settingsMapByName[nameOrUid] ?? this.settingsMapById[nameOrUid];
```

Comment:
Is there a formatting option that lets us put these on their own lines? 
 it took me a while to read how they are different :)
```
return 
   this.settingsMapByUid[nameOrUid] ??  // UID
   this.settingsMapByName[nameOrUid] ??  // Name
   this.settingsMapById[nameOrUid] // Internal ID
```

Reply:
I prefer the way you prefer but I do not think that is in our prettier configuration